ts/Extensions/PriceIndication.ts

```diff
@@ -144,9 +146,12 @@ addEvent(Series, 'afterRender', function (): void {
             crop;
 
         if (lastPrice && lastPrice.enabled) {
-
             yAxis.crosshair = yAxis.options.crosshair = seriesOptions.lastPrice;
```

Comment:
Is this line still necessary? Below, we do the merge, so maybe we can remove it? :)